pkg/operation/botanist/component/apiserverproxy/apiserver_proxy.go

```diff
@@ -0,0 +1,588 @@
+									ProbeHandler: corev1.ProbeHandler{
+										HTTPGet: &corev1.HTTPGetAction{
+											Path: "/ready",
+											Port: intstr.FromInt(int(a.values.AdminPort)),
+										},
+									},
+									InitialDelaySeconds: 1,
+									PeriodSeconds:       2,
+									SuccessThreshold:    1,
+									TimeoutSeconds:      1,
+								},
+								LivenessProbe: &corev1.Probe{
+									ProbeHandler: corev1.ProbeHandler{
+										HTTPGet: &corev1.HTTPGetAction{
+											Path: "/ready",
+											Port: intstr.FromInt(int(a.values.AdminPort)),
+										},
+									},
+									InitialDelaySeconds: 1,
+									PeriodSeconds:       10,
+									SuccessThreshold:    1,
+									TimeoutSeconds:      1,
+									FailureThreshold:    3,
+								},
+								Ports: []corev1.ContainerPort{
+									{
+										Name:          "metrics",
+										ContainerPort: a.values.AdminPort,
+										HostPort:      a.values.AdminPort,
+									},
+								},
+								VolumeMounts: []corev1.VolumeMount{
+									{
+										Name:      proxyConfigVolume,
+										MountPath: "/etc/apiserver-proxy",
+									},
+									{
+										Name:      adminUDSVolume,
+										MountPath: "/etc/admin-uds",
+									},
+								},
+							},
+						},
+						Volumes: []corev1.Volume{
+							{
+								Name: proxyConfigVolume,
+								VolumeSource: corev1.VolumeSource{
+									ConfigMap: &corev1.ConfigMapVolumeSource{
+										LocalObjectReference: corev1.LocalObjectReference{
+											Name: configMap.Name,
+										},
+									},
+								},
+							},
+							{
+								Name: adminUDSVolume,
+								VolumeSource: corev1.VolumeSource{
+									EmptyDir: &corev1.EmptyDirVolumeSource{},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+	)
+
+	if a.values.PodMutatorEnabled {
+		clusterCASecret, found := a.secretsManager.Get(v1beta1constants.SecretNameCACluster)
+		if !found {
+			return nil, fmt.Errorf("secret %q not found", v1beta1constants.SecretNameCACluster)
+		}
+		var (
+			failurePolicy      = admissionregistrationv1.Ignore
+			matchPolicy        = admissionregistrationv1.Exact
+			reinvocationPolicy = admissionregistrationv1.NeverReinvocationPolicy
+			scope              = admissionregistrationv1.AllScopes
+			sideEffects        = admissionregistrationv1.SideEffectClassNone
+
+			mutatingWebhook = &admissionregistrationv1.MutatingWebhookConfiguration{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: mutatingWebhookName,
+					Annotations: map[string]string{
+						"networking.gardener.cloud/description": `This webhook adds KUBERNETES_SERVICE_HOST
+environment variable to all containers and init containers matched by it.`,
+					},
+					Labels: utils.MergeStringMaps(
+						configMap.GetLabels(),
+						map[string]string{v1beta1constants.LabelExcludeWebhookFromRemediation: "true"}),
+				},
+				Webhooks: []admissionregistrationv1.MutatingWebhook{
+					{
+						AdmissionReviewVersions: []string{"v1beta1"},
```

Comment:
cc @ScheererJ - is the apiserver-proxy not capable of also handling the `v1` review version? If so, we should add support for it (I expect `v1beta1` to be dropped sooner or later). Otherwise, please add `v1` here as well.

Reply:
As far as I can tell, `apiserver-proxy` supports `v1` since https://github.com/gardener/apiserver-proxy/commit/5f981c8a2f867fbd35f90de5c3be0bc30c6c8cb5 as it updates controller-runtime to [0.7.0](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.7.0), which switched admission types to v1. Therefore, I would even state that `apiserver-proxy` actually only supports `v1`.

I will update the example in the `apiserver-proxy` repository accordingly.

Reply:
https://github.com/gardener/apiserver-proxy/pull/35

Reply:
TLDR:
```suggestion
						AdmissionReviewVersions: []string{"v1"},
```

Reply:
done 😄